src/core/meshcop/leader_mtd.hpp

```diff
@@ -44,6 +44,11 @@ class Leader
 public:
     Leader(ThreadNetif &) { }
     ThreadError SendDatasetChanged(const Ip6::Address &) { return kThreadError_NotImplemented; }
+    ThreadError SetDelayTimerMinimal(uint32_t aDelayTimerMinimal) {
```

Comment:
nit: if you simply declare this as `SetDelayTimerMinimal(uint32_t)`, then you can avoid needing to include the line below.